workflows/wdl/starfish.wdl

```diff
@@ -0,0 +1,100 @@
+
+task process_field_of_view {
+
+    String experiment
+    Int field_of_view
+    String recipe_file
```

Comment:
```suggestion
    File recipe_file
```

If you declare this a file, the WDL runner will localize it for you, then you can delete the wget line below.

Reply:
This does not work. The file has to be local already https://github.com/openwdl/wdl/blob/master/versions/1.0/SPEC.md#task-inputs